spec/openapi.yaml

```diff
@@ -580,6 +580,32 @@ paths:
                     type: array
                     items:
                       $ref: "#/components/schemas/Customer"
+    post:
+      tags:
+        - Customers
+      summary: Create
+      description: Create a new customer.
+      operationId: createCustomer
+      parameters:
+        - $ref: "#/components/parameters/version"
+      requestBody:
+        content:
+          application/json:
+            schema:
+              properties:
+                email:
+                  description: Email address
+                  type: string
+                  format: email
```

Comment:
There are also `firstName` and `lastName` properties. And because there are plans to make these required in the storefront, I would suggest to already document them as being required here to avoid BC breaks.